gcloud/streaming/test_transfer.py

```diff
@@ -1785,6 +1785,18 @@ def test__send_chunk_w_total_size_stream_exhausted(self):
         self.assertEqual(end, SIZE)
 
 
+def _email_chunk_parser():
+    import six
+    if six.PY3:  # pragma: NO COVER  Python3
+        from email.parser import BytesParser  # pylint: disable=E0611
```

Comment:
@tseaver Please replace with

```python
        # pylint: disable=no-name-in-module
        from email.parser import BytesParser
        # pylint: enable=no-name-in-module
```

logic being that future maintainers (including future us) won't need to Google for what `E0611` is.